tests/main/interfaces-content-mimic/task.yaml

```diff
@@ -21,11 +22,17 @@ execute: |
     # entries that should be correctly replicated inside the mimic.
     
     check_existing_files() {
+        #shellcheck disable=SC2016
```

Comment:
This is all correct, it's too bad there's no way to express this in a shellcheck happy way, apart from disabling this check.